invokeai/app/api_app.py

```diff
@@ -142,10 +144,11 @@ def overridden_redoc():
 
 
 # Must mount *after* the other routes else it borks em
-app.mount(
-    "/", StaticFiles(directory="invokeai/frontend/web/dist", html=True), name="ui"
-)
-
+app.mount("/", 
+          StaticFiles(directory=Path(web_dir.__path__[0],"dist"), 
```

Comment:
Same comment re: `web_dir` not existing

Reply:
See above. There was a missing import statement.